### PR TITLE
Update related documents page for JakartaEE

### PR DESCRIPTION
**Pull request by aguibert**
For issue #38 

It looks like we don't have specs published anywhere for the individual projects yet (chicken and the egg problem?) so I've just left placeholders for where the links need to go eventually.

### CODE REVIEW

**Comment by bshannon** [comment]
Spec URLs are described[here](https://www.eclipse.org/lists/jakarta.ee-spec/msg00035.html).

File: specification/src/main/asciidoc/platform/RelatedDocuments.adoc [hunk 1]
```diff
@@ -4,149 +4,85 @@ This specification refers to the following
 documents. The terms used to refer to the documents in this
 specification are included in parentheses.
 
- _Java™ Platform, Enterprise Edition
-Specification Version 8_ (this specification). Available at
-_http://jcp.org/en/jsr/detail?id=366_ .
+Jakarta™ EE Platform Specification Version 8. Available at: <link needed>.
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/RelatedDocuments.adoc]: Following the link in @bshannon's comment, the missing ink should be along the lines of http://jakarta.ee/specifications/full-platform/8.0.1.  The tricky part will be to figure out the exact version that will be used.  But, take your best guess for now.

Comment (kwsutter) [inline specification/src/main/asciidoc/platform/RelatedDocuments.adoc]: Also, why did you remove all of the italics?  The underscores before and after each entry.

File: specification/src/main/asciidoc/platform/RelatedDocuments.adoc [hunk 2]
```diff
@@ -4,149 +4,85 @@ This specification refers to the following
...
+Jakarta™ Annotations Specification, Version 1.3. Available at: <link needed>.
 
- _Java™ Servlet Specification, Version 4.0_
-(Servlet specification). Available at
-_http://jcp.org/en/jsr/detail?id=369_ .
+Jakarta™ Servlet Specification, Version 4.0. Available at: <link needed>.
 
- _JDBC™ 4.2 API_ (JDBC specification). Available
-at _http://jcp.org/en/jsr/detail?id=221_ .
+JDBC™ 4.2 API (JDBC specification). Available at: http://jcp.org/en/jsr/detail?id=221
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/RelatedDocuments.adoc]: Again, don't remove the italics.   This is through out the PR, so I won't repeat myself.  :-)

File: specification/src/main/asciidoc/platform/RelatedDocuments.adoc [hunk 3]
```diff
@@ -4,149 +4,85 @@ This specification refers to the following
...
-_http://docs.oracle.com/javase/8/docs/technotes/guides/jndi/index.html[]http://docs.oracle.com/javase/8/docs/technotes/guides/jndi/index.html._
+Java™ Naming and Directory Interface 1.2 Specification (JNDI specification). Available at: http://docs.oracle.com/javase/8/docs/technotes/guides/jndi/index.html
 
- _Java™ Message Service, Version 2.0_ (JMS
-specification). Available at _http://jcp.org/en/jsr/detail?id=343_ .
+Jakarta Messaging Specification, Version 2.0. Available at: <link needed>.
 
- _Java™ Transaction API, Version 1.2_ (JTA
-specification). Available at _http://jcp.org/en/jsr/detail?id=907_ .
+Jakarta™ Transaction API Specification, Version 1.2. Available at: <link needed>.
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/RelatedDocuments.adoc]: Jakarta Transactions (no API).

File: specification/src/main/asciidoc/platform/RelatedDocuments.adoc [hunk 4]
```diff
@@ -4,149 +4,85 @@ This specification refers to the following
...
 
- _Java™ Transaction Service, Version 1.0_ (JTS
-specification). Available at
-_http://www.oracle.com/technetwork/java/javaee/jts-spec095-1508547.pdf_
-.
+Java™ Transaction Service, Version 1.0_ (JTS specification). Available at: http://www.oracle.com/technetwork/java/javaee/jts-spec095-1508547.pdf
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/RelatedDocuments.adoc]: Reinstate the italics and we'll be fine with JTS as is.

File: specification/src/main/asciidoc/platform/RelatedDocuments.adoc [hunk 5]
```diff
@@ -4,149 +4,85 @@ This specification refers to the following
...
+Jakarta™ Interceptors Specification, Version 1.2. Available at: <link needed>.
 
-Contexts and Dependency Injection for the Java
-EE Platform 2.0 (CDI specification). Available at
-_http://jcp.org/en/jsr/detail?id=365_ .
+Jakarta™ Contexts and Dependency Injection Specification, Version 2.0. Available at: <link needed>.
 
-Dependency Injection for Java 1.0 (DI
-specification). Available at _http://jcp.org/en/jsr/detail?id=330_ .
+Dependency Injection for Java 1.0 (DI specification). Available at http://jcp.org/en/jsr/detail?id=330
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/RelatedDocuments.adoc]: I believe DI is going to be produced as part of CDI...  For now, this is okay, but we'll have to watch this space.

File: specification/src/main/asciidoc/platform/RelatedDocuments.adoc [hunk 6]
```diff
@@ -220,39 +156,38 @@ instance and is independent of the number of Java virtual machines,
...
 
-[.footnoteNumber]# 6.# [[a3653]]See the JSF
-specification section “JSF Managed Classes and Java EE Annotations” for
+[.footnoteNumber]# 6.# [[a3653]]See the Jakarta™ Server Faces
+specification section Jakarta™ Server Faces Managed Classes and Jakarta Annotations” for
 a list of these managed classes.
 
-[.footnoteNumber]# 7.# [[a3654]]Resource and CDI
-injection is supported only in JAX-RS components managed by CDI.
+[.footnoteNumber]# 7.# [[a3654]]Resource and Jakarta™ Contexts and Dependency Injection
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/RelatedDocuments.adoc]: I would suggest we generalize this a bit...  "Resource, context, and dependency injection is supported..."

File: specification/src/main/asciidoc/platform/RelatedDocuments.adoc [hunk 7]
```diff
@@ -220,39 +156,38 @@ instance and is independent of the number of Java virtual machines,
...
+injection is supported only in Jakarta™ RESTful Web Services components managed by Jakarta™ Contexts and Dependency Injection.
 
-[.footnoteNumber]# 8.# [[a3655]]Interceptors cannot be
+[.footnoteNumber]# 8.# [[a3655]]Jakarta™ Interceptors cannot be
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/RelatedDocuments.adoc]: Leave this with just Inteceptors. Drop Jakarta.

File: specification/src/main/asciidoc/platform/RelatedDocuments.adoc [hunk 8]
```diff
@@ -220,39 +156,38 @@ instance and is independent of the number of Java virtual machines,
...
-refer to classes that become managed beans per the rules in the CDI
+refer to classes that become managed beans per the rules in the Jakarta™ Contexts and Dependency Injection
 specification, thus excluding managed beans declared using the
-_ManagedBean_ annotation as well as EJB session beans, both of which
-would be managed beans even in the absence of CDI.
+_ManagedBean_ annotation as well as Jakarta™ Enterprise Beans session beans, both of which
+would be managed beans even in the absence of Jakarta™ Contexts and Dependency Injection.
 
-[.footnoteNumber]# 11.# [[a3658]]Interceptors cannot
+[.footnoteNumber]# 11.# [[a3658]]Jakarta Interceptors cannot
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/RelatedDocuments.adoc]: Drop Jakarta in this case.

File: specification/src/main/asciidoc/webprofile/RelatedDocuments.adoc [hunk 9]
```diff
@@ -4,81 +4,48 @@ This specification refers to the following
 documents. The terms used to refer to the documents in this
 specification are included in parentheses.
 
- _Java™ Platform, Enterprise Edition
-Specification Version 8_ . Available at
-_http://jcp.org/en/jsr/detail?id=366_ .
+Jakarta™ EE Platform Specification Version 8. Available at: <link needed>.
```
Comment (kwsutter) [inline specification/src/main/asciidoc/webprofile/RelatedDocuments.adoc]: Similar comments as above for Platform version of this doc.

File: specification/src/main/asciidoc/platform/RelatedDocuments.adoc (hunk 1 above)
Comment (bshannon) [reply specification/src/main/asciidoc/platform/RelatedDocuments.adoc]: Italics would be the normal bibliographic reference style.

File: specification/src/main/asciidoc/platform/RelatedDocuments.adoc (hunk 5 above)
Comment (bshannon) [reply specification/src/main/asciidoc/platform/RelatedDocuments.adoc]: Yes, the CDI project will be producing the DI spec.

File: specification/src/main/asciidoc/platform/RelatedDocuments.adoc (hunk 1 above)
Comment (aguibert) [reply specification/src/main/asciidoc/platform/RelatedDocuments.adoc]: I can put them back, I didn't realize they were for italics. I had assumed they were some invalid chars that got copied over from the original spec document (perhaps a PDF or something)

**Comment by aguibert** [comment]
@kwsutter @bshannon all comments have been addressed -- ready for re-review
